### ISSUE TITLE
Add all HTML Semantic Elements

### ISSUE THREAD

**Issue by 463** [menatwork/semantic_html5#41]
It would be fine to add all defined standard elements.

[https://www.w3schools.com/html/html5_semantic_elements.asp](url)

**Comment by kikmedia** [comment]
Wouldn't you be able to add the desired elements in an own config.php file? Probably see contao/config/config.php

**Comment by davidmaack** [comment]
Yes, you could simply add any HTML-Tag by extending the array of elements with your own config file. Is there an actual need to define the elements in the Backend maybe?
